src/messaging/Message.ts

```diff
@@ -218,10 +218,34 @@ interface IMessageBodyBase {
   type: MessageBodyType
 }
 
+interface ICostBreakdown {
+  tax: number
+  net: number
+  gross: number
+}
```

Comment:
Created a new Cost breakdown as I felt it fitted to have an object that could be versionable in the future since we can expand on this and break down the data further as a potential addition of costs.